pkgs/development/libraries/libnotify/default.nix

```
@@ -1,20 +1,18 @@
-{ stdenv, fetchurl, pkgconfig, autoreconfHook
+{ stdenv, fetchurl, meson, ninja, pkgconfig
 , glib, gdk_pixbuf, gobject-introspection }:
 
 stdenv.mkDerivation rec {
-  ver_maj = "0.7";
-  ver_min = "7";
-  name = "libnotify-${ver_maj}.${ver_min}";
+  pname = "libnotify";
+  version = "0.7.8";
 
   src = fetchurl {
-    url = "mirror://gnome/sources/libnotify/${ver_maj}/${name}.tar.xz";
-    sha256 = "017wgq9n00hx39n0hm784zn18hl721hbaijda868cm96bcqwxd4w";
+    url = "mirror://gnome/sources/libnotify/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "1371csx0n92g60b5dmai4mmzdnx8081mc3kcgc6a0xipcq5rw839";
   };
 
-  # disable tests as we don't need to depend on gtk+(2/3)
-  configureFlags = [ "--disable-tests" ];
+  mesonFlags = [ "-Dtests=false" "-Ddocbook_docs=disabled" "-Dgtk_doc=false" ];
 
-  nativeBuildInputs = [ pkgconfig autoreconfHook gobject-introspection ];
+  nativeBuildInputs = [ meson ninja pkgconfig gobject-introspection ];
   buildInputs = [ glib gdk_pixbuf ];
 
```

As per https://github.com/NixOS/nixpkgs/issues/36150:

```suggestion

  passthru = {
    updateScript = gnome3.updateScript {
      packageName = pname;
      versionPolicy = "none";
    };
  };

```